docs/reference/commands/cluster.rst

```diff
@@ -0,0 +1,210 @@
+===========
+``cluster``
+===========
+
+Manage the MicroCeph cluster
+
+Usage:
+
+.. code-block:: none
+
+   microceph cluster [options]
+   microceph cluster [command]
+
+Available commands:
+
+.. code-block:: none
+
+  add         Generates a token for a new server
+  bootstrap   Sets up a new cluster
+  config      Manage Ceph Cluster configs
+  join        Joins an existing cluster
+  list        List servers in the cluster
+  migrate     Migrate automatic services from one node to another
+  remove      Removes a server from the cluster
+  sql         Runs a SQL query against the cluster database
+
+
+Global options:
+
+.. code-block:: none
+
+   -d, --debug       Show all debug messages
+   -h, --help        Print help
+       --state-dir   Path to store state information
+   -v, --verbose     Show all information messages
+       --version     Print version number
+
+``add``
+-------
+
+Generates a token for a new server
+
+Usage:
+
+.. code-block:: none
+
+  microceph cluster add <NAME> [flags]
```

Comment:
Let's align the first command string directly under 'code'.

So here, 'm' should be directly under 'c'.

Let's do this everywhere.